microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -344,3 +344,20 @@ groups:
           title: "Data Service Connection Limit Exceeded (node: {{ $labels.hostname }})"
           description: Client connections have been rejected because the maximum number of connections has been exceeded.
           remediation: Ensure your applications are appropriately closing their Couchbase connections.
+
+      - alert: CB90073-droppedDCPMutations
+        expr: |
+          count_over_time({file="memcached.log"} |= "Dropping mutation" [1h]) >0
+        labels:
+          job: couchbase_fluent_bit
+          kind: cluster
+          severity: critical
+          health_check_id: CB90073
+          health_check_name: droppedDCPMutations
+          cluster: '{{ $labels.couchbase_cluster }}'
+          node: '{{ $labels.hostname }}'
+        annotations:
+          title: "Dropped DCP Mutations on cluster (cluster: {{ $labels.couchbase_cluster }})"
+          description: Dropped DCP mutations found in memcached.log.
+            This can be an indication of a Couchbase Server Issue  (MB-41255) which was fixed in versions: 6.0.5, 6.5.2 and 6.6.1. This issue can affect data integrity.
```

Comment:
Because of the colon, YAML is parsing this as a key-value pair with a key of `This can be an indication of a Couchbase Server Issue  (MB-41255) which was fixed in versions` and a value of `6.0.5, 6.5.2 and 6.6.1. This issue can affect data integrity.`. Might be better if you switch to using multiline string syntax:

```yaml
description: |
  Dropped DCP mutations found in memcached.log
  This can be an indication of a Couchbase Server Issue  (MB-41255) which was fixed in versions: 6.0.5, 6.5.2 and 6.6.1. This issue can affect data integrity.
```

(though I'm not actually sure how Prometheus will handle the multiline strings - will it just glue them together into one line, or will it put in a newline)

Reply:
Fixed :)